modules/mod_video_embed/mod_video_embed.erl

```diff
@@ -343,10 +344,11 @@ event(#submit{message={add_video_embed, EventProps}}, Context) ->
                 {video_embed_service, EmbedService},
                 {video_embed_code, EmbedCode}
             ],
-            case m_rsc:update(Id, Props, Context) of
-                {ok, _} ->
-                    z_render:wire([{dialog_close, []} | Actions], Context);
-                {error, _} ->
+
+            try
+                {ok, _} = m_rsc:update(Id, Props, Context)
+            catch
+                _ ->
                     z_render:growl_error("Could not update the page with the new embed code.", Context)
```

Comment:
Line 352 is too long:                     z_render:growl_error("Could not update the page with the new embed code.", Context).